From a73ec5377c9413e25f1de7820cd7b9a66302565c Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:05:58 -0400
Subject: [tools] perf/evsel: Update function names in debug messages

Message-id: <1410181752-25631-114-git-send-email-jolsa@redhat.com>
Patchwork-id: 91766
O-Subject: [PATCH RHEL7.1 BZ1134356 113/307] perf evsel: Update function names in debug messages
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit a33f6efc43cb71795bbdaf0251544ff3edbfdb2f
Author: Ramkumar Ramachandra <artagnon@gmail.com>
Date: Tue Mar 18 15:10:42 2014 -0400

description
===========
perf_event_open() was renamed to sys_perf_event_open(); update the debug
messages to reflect this.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/evsel.c b/tools/perf/util/evsel.c
index 26b67b1..5c28d82 100644
--- a/tools/perf/util/evsel.c
+++ b/tools/perf/util/evsel.c
@@ -1023,7 +1023,7 @@ retry_sample_id:
 
    group_fd = get_group_fd(evsel, cpu, thread);
 retry_open:
-   pr_debug2("perf_event_open: pid %d  cpu %d  group_fd %d  flags %#lx\n",
+   pr_debug2("sys_perf_event_open: pid %d  cpu %d  group_fd %d  flags %#lx\n",
       pid, cpus->map[cpu], group_fd, flags);
 
    FD(evsel, cpu, thread) = sys_perf_event_open(&evsel->attr,
@@ -1032,7 +1032,7 @@ retry_open:
              group_fd, flags);
    if (FD(evsel, cpu, thread) < 0) {
     err = -errno;
-    pr_debug2("perf_event_open failed, error %d\n",
+    pr_debug2("sys_perf_event_open failed, error %d\n",
        err);
     goto try_fallback;
    }
-- 
1.7.1